ci_support/run_docker_build.sh

```diff
@@ -39,9 +38,16 @@ echo "$config" > ~/.condarc
 # A lock sometimes occurs with incomplete builds. The lock file is stored in build_artefacts.
 conda clean --lock
 
-conda update --yes --all
-conda install --yes conda-build
-conda info
+conda install --yes --quiet conda-forge-build-setup
+source run_conda_forge_build_setup
+
+
+# Install the yum requirements defined canonically in the
+# "recipe/yum_requirements.txt" file. After updating that file,
+# run "conda smithy rerender" and this line be updated
+# automatically.
+yum install -y java-1.8.0-openjdk-devel unzip zip zlib-devel rsync
```

Comment:
rerender? Then it is fine. (I saw the lines and assumed they were in `build.sh` but now I see where they are.)